.github/tests/it/client/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.21-alpine AS builder
+FROM golang:1.22.1-alpine AS builder
```

Comment:
The version in the Dockerfiles used for the integration tests should be parametrized, but it's out of scope for this PR.